### PR TITLE
Slides: Fix no shadow on cards in swiper container

### LINKED ISSUE #3135: [Enhancement] No shadow on Slider cards
- [x] I have written a [good issue](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Issue) 

## Describe the enhancement
The default cards has a drop shadow. When cards are used in Slider component, the card shadow is hidden by the "swiper-container" (overflow = hidden). The result is that the cards does not look right.

We still need the "swiper-container" to hide the cards that are not in view.

## Describe the solution you'd like
Find a new way to hide cards not in view, but keep the card shadow visible.

## Have you considered any alternatives?
No

## Are there any additional context?
**Card shadow hidden:**
Skærmbillede 2023-06-29 kl  10 35 19

**Card shadow NOT hidden:**
Skærmbillede 2023-06-29 kl  10 43 08

<hr />

## Checklist:

The following tasks should be carried out in sequence in order to follow [the process of contributing](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#the-process-of-contributing) correctly.

### Refinement

- [ ] Request that the issue is [UX refined](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#ux-refinement); do not proceed until this is done.
- [ ] Request that the issue is [tech refined](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#tech-refinement); do not proceed until this is done.

### Implementation 
The contributor who wants to implement this issue should: 

- [ ] Make sure you have read: "[Before you get coding](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#before-you-get-coding)".
- [ ] Signal to others you are working on the issue by assigning yourself.
- [ ] Create a branch from the [develop branch](https://github.com/kirbydesign/designsystem/tree/develop) following our [branch naming convention](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Branch). 
- [ ] Publish a WIP implementation to Github as a draft PR and ask for feedback. 
- [ ] Make sure you have implemented tests following the guidelines in: "[The good: Test](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Test)".
- [ ] Update the [cookbook](https://cookbook.kirby.design) with examples and showcases.

### Review
Once the issue has been implemented and is ready for review:

- [ ] Do a [self-review](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Self-review). 
- [ ] Create a pull-request. If you created a draft PR during implementation you can just mark that as "ready for review".

### PR DESCRIPTION
**Pull request by AgreSanGit**
## Which issue does this PR close?

This PR closes #3135 
## What is the new behavior?

Introduce padding in SwiperContainer with negative margin, to make shadow on cards visible in slides component, and also only showing the card when it is fully visible on desktop.

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## Checklist:

The following tasks should be carried out in sequence in order to follow [the process of contributing](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#the-process-of-contributing) correctly.

### Reminders
- [ ] Make sure you have implemented tests following the guidelines in: "[The good: Test](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Test)".
- [ ] Make sure you have updated the cookbook with examples and showcases (for bug fixes, enhancements & new components).

### Review  
- [ ] Determine if your changes are a fix, feature or breaking-change, and add the matching label to your PR. If it is tooling, dependency updates or similar, add ignore-for-release.
- [ ] Do a [self-review](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Self-review).
- [ ] Request that the changes are code-reviewed 
- [ ] Request that the changes are [UX reviewed](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#ux-review) (only necessary if your PR introduces visual changes)

When the pull request has been approved it will be merged to `develop` by Team Kirby.

### CODE REVIEW

File: libs/designsystem/slide/src/slides.component.scss [hunk 1]
```diff
@@ -70,6 +72,11 @@ $bullet-inactive-opacity: 0.2;
     display: none;
   }
 
+  swiper-container {
+    padding: utils.size('m') utils.size('s');
+    margin: calc(-1 * utils.size('m')) calc(-1 * utils.size('s'));
```
Comment (jakobe) [inline libs/designsystem/slide/src/slides.component.scss]: Consider simplifying as calculations are built into Sass — and you can do just negate the values:
```suggestion
    margin: -utils.size('m') -utils.size('s');
```

File: libs/designsystem/slide/src/slides.component.scss [hunk 2]
```diff
@@ -43,7 +43,9 @@ $bullet-inactive-opacity: 0.2;
 
   .buttons {
     display: none;
-    z-index: utils.z('default');
+
```
Comment (jakobe) [inline libs/designsystem/slide/src/slides.component.scss:46]: Consider readability:
```suggestion
```

**Review by jakobe — COMMENTED** [review]
Nice work 👌🏻 

2 small comments to improve readability of the code 😊

File: libs/designsystem/slide/src/slides.component.scss (hunk 1 above)
Comment (AgreSanGit) [reply libs/designsystem/slide/src/slides.component.scss]: That's nice to know!

**Review by jakobe — DISMISSED** [review]
🤩

**Review by jakobe — APPROVED** [review]
:shipit:

**Pull request merged**